infrastructure/database/templates/03_setup_tables.template.sql

```diff
@@ -215,7 +215,9 @@ CREATE TABLE private.tree_devices (
     customer_id INTEGER REFERENCES private.customers(id),
     device_id  BIGINT,
     site_id BIGINT,
-    PRIMARY KEY(tree_id, customer_id, device_id, site_id)
+    valid_from_date DATE,
+    valid_to_date DATE,
+    PRIMARY KEY(tree_id, customer_id, device_id)
```

Comment:
@birdMarcus 
`site_id` fällt weg?

Reply:
als key ja. Ist nur ne zusätliche Info, die ich für's Matching brauche, die wir sonst nicht in der DB haben. Ist nicht ideal denormalisiert. Aber da jetzt noch ne Extra Tabelle für fand auch unnötig. Sind ja nur paar 100 Einträge, daher auch nicht zu ineffizient vom Speicher her.